python/src/TensorizedUniVariateFunctionFactory_doc.i.in

```diff
@@ -0,0 +1,36 @@
+%feature("docstring") OT::TensorizedUniVariateFunctionFactory
+"Base class for tensorized multivariate functions.
+
+Available constructors:
+        TensorizedUniVariateFunctionFactory(*functions*)
+
+        TensorizedUniVariateFunctionFactory(*functions, enumerateFunction*)
+
+Parameters
+----------
+functions : list of :class:`~openturns.UniVariateFunctionFamily`
+    List of univariate function factories.
+enumerateFunction : :class:`~openturns.EnumerateFunction`
+    Associates to an integer its multi-index image in the :math:`\Nset^d`
+    dimension, which is the dimension of the basis. This multi-index represents
+    the collection of degrees of the univariate polynomials.
+
+Notes
+-----
+TensorizedUniVariateFunctionFactory is a particular case of implementation of
+the :class:`~openturns.Basis` in the case of linear regression. 
```

Comment:
A formula is also mandatory here: if enum is the enumerate function, then:
\Phi_n(x_1,\dots,x_d)=\prod_{i=1}^d\phi^i_{enum(n)_i}(x_i)
where (\phi^i_k)_{k\in\Nset} is the univariate basis associated to the component x_i and enum(n)_i is the ith component of the multi-index enum(n)